api/citations/utils.py

```diff
@@ -48,18 +48,15 @@ def render_citation(node, style='apa'):
 
     bib_source = CiteProcJSON(data)
 
-    bib_style = CitationStylesStyle(os.path.join(CITATION_STYLES_PATH, style), validate=False)
+    path = os.path.join(BASE_PATH, 'static', 'bluebook') if 'bluebook' in style else os.path.join(CITATION_STYLES_PATH, style)
```

Comment:
I have a feeling that we're going to want to do this again. Should we just make an array of objects with name and filename so it'll be easy in the future?